src/block/posts/util.js

```diff
@@ -199,7 +199,19 @@ export const generateRenderPostItem = attributes => {
 
 		// Trim the excerpt.
 		let excerptString = postExcerptStackable.split( ' ' )
-		if ( excerptString.length > ( excerptLength || 55 ) ) {
+
+		//Checks if there are CJK characters present
+		const regexCJK = /[\u3040-\u30ff\u3400-\u4dbf\u4e00-\u9fff\uf900-\ufaff\uff66-\uff9f]/
+		if ( postExcerptStackable.match( regexCJK ) ) {
+			// Extract the string inbetween the tags and newline.
+			const tempExcerptString = postExcerptStackable.match( '<p>(.*)<\/p>\\n' )
+
+			if ( tempExcerptString[ 1 ].length > ( excerptLength || 55 ) ) {
+				excerptString = tempExcerptString[ 1 ].substring( 0, excerptLength === '' ? 55 : excerptLength ) + '...'
+			} else {
+				excerptString = tempExcerptString[ 1 ]
+			}
```

Comment:
The `tempExcerptString[ 1 ]` errors out & I'm not sure why we need `tempExcerptString`, we can just use this directly:
```suggestion
			if ( postExcerptStackable.length > ( excerptLength || 55 ) ) {
				excerptString = postExcerptStackable.substring( 0, excerptLength === '' ? 55 : excerptLength ) + '...'
			} else {
				excerptString = postExcerptStackable
			}
```

Reply:
`const tempExcerptString = postExcerptStackable.match( '<p>(.*)<\/p>\\n' )` returns an array and the 1st element is not the match. Though I am not sure if this is the best way to do it.